src/spec/index.js

```diff
@@ -4,25 +4,22 @@ import Info from './info';
 import Server from './server';
 import Spec from './spec';
 
-function buildSpec({ info, openapiVersion, operations, server, ...more }) {
-    const spec = new Spec({
+function buildSpec({ info, operations, server, ...options }) {
```

Comment:
Building a spec just builds a `Spec`. 

This is a subtle API change (this function used to build the Spec into an OpenAPI JSON doc... but nothing depends on this behavior yet, so it's kosher to change.